Exdi/exdigdbsrvsample/ExdiGdbSrvSample/ExdiGdbSrvSample.vcxproj

```diff
@@ -238,6 +238,11 @@
       <Command>
       </Command>
     </PreBuildEvent>
+    <PostBuildEvent>
+      <Command>copy "$(SolutionDir)\GdbSrvControllerLib\exdiConfigData.xml" "$(TargetDir)\exdiConfigData.xml"

```

Comment:
You can remove this PostBuildEvent, I already added a copy on the item level for those two XML files.